unified-codes/tools/dockerfiles/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: '3'
+services:
+  legacy-api:
+    container_name: legacy-api
+    build: 
+      context: ../..
+      dockerfile: tools/dockerfiles/legacy-api.Dockerfile
+    image: unified-codes/legacy-api:latest
+    ports:
+      - "3000:3000"
+    environment:
+      - NX_DGRAPH_ENDPOINT=http://dgraph-standalone:8080
+    volumes:
+      - ../../dist/apps/legacy-api:/usr/unified-codes/dist/apps/legacy-api
+  data-service:
+    build: 
+      context: ../..
+      dockerfile: tools/dockerfiles/data-service.Dockerfile
+    container_name: data-service
+    image: unified-codes/data-service:latest
+    ports:
+      - "4000:4000"
+    environment:
+      - NX_DGRAPH_ENDPOINT=http://dgraph-standalone:8080
+      - NX_AUTHENTICATION_URL=http://keycloak:9990/auth/realms/unified-codes
+    volumes:
+      - ../../dist/apps/data-service:/usr/unified-codes/dist/apps/data-service
+  dgraph-standalone:
+    container_name: dgraph-standalone
+    image: dgraph/standalone:v20.03.0
+    ports: 
+      - "8080:8080"
+      - "9080:9080"
+      - "8000:8000"
+    volumes:
+      - ~/dgraph:/dgraph
+  keycloak:
+    container_name: keycloak
+    image: jboss/keycloak
+    ports: 
+      - "9990:8080"
+    environment:
+      - KEYCLOAK_USER=admin
+      - KEYCLOAK_PASSWORD=admin 
```

Comment:
Does this retire some of our existing scripts for spinning up these containers?

Reply:
Hmm it could potentially, though in its current state it will also spin up legacy-api and data-service in a non-debuggable way.

Maybe that's fine and we can manually stop them when we want to start debugging? 🤷‍♀️ 
Alternatively we could define a local dev config that just spins up keycloak and dgraph?

Reply:
+1 for a local dev config, if that's not too hard to do!

Reply:
Cool, might do that in a separate PR! Seems like it shouldn't be too difficult